apps/backend/src/auth/canModifyProject.guard.ts

```diff
@@ -0,0 +1,29 @@
+import { AuthGuard } from '@nestjs/passport';
+import { ExecutionContext, Injectable } from '@nestjs/common';
+import { Reflector } from '@nestjs/core';
+import { JwtService } from '@nestjs/jwt';
+
+@Injectable()
+export class CanModifyProjectGuard extends AuthGuard('local') {
+  constructor(private readonly _reflector: Reflector, private jwtService: JwtService) {
+    super();
+  }
+
+  canActivate(context: ExecutionContext) {
+    /*
+        CHECKINGS :
+        - IS JWT VALID ?
+        - IS THE ID IN JWT THE SAME AS IN THE SENT OBJECT ?
+     */
+    const request = context.switchToHttp().getRequest();
+    const { headers } = request;
+    const headerString = headers.authorization.split(' ');
+    const jwtData = this.jwtService.decode(headerString[1]) as { [key: string]: never };
+    // In case of incoherent jwt.
```

Comment:
Checking the JWT token validity could be done in a dedicated method.

Reply:
Indeed. As we saw previously, I am waiting for more guards with Jwt implementation (might be done very soon).